app/src/main/res/values-pt/strings.xml

```diff
@@ -60,4 +60,33 @@
     <string name="d_testConnection_urlSuggestion_acceptButton">Usar sugerido</string>
     <string name="d_testConnection_urlSuggestion_declineButton">Usar meu</string>
     <string name="d_testConnection_urlSuggestion_cancelButton">Cancelar</string>
-    </resources>
+    <string name="checkCredentials_progressDialogMessage">Testando o acesso à API</string>
+    <string name="connectionWizard_providerSelection_next">Seguinte</string>
+    <string name="connectionWizard_providerSelection_prev">Pular</string>
+    <string name="connectionWizard_providerSelection_provider_general_name">Outro</string>
+    <string name="connectionWizard_provider_framabag_titleMessage">Configuração de Framabag</string>
+    <string name="connectionWizard_summary_toastMessage">Configuração completada</string>
+    <string name="connectionWizard_welcome_next">Seguinte</string>
+    <string name="connectionWizard_welcome_prev">Pular</string>
+    <string name="connectionWizard_welcome_titleMessage">Ben-vindo ao wallabag.</string>
+    <string name="content_estimatedReadingTime">Tempo estimado de leitura: %s min</string>
+    <string name="d_checkCredentials_errorMessage_unknown">Error desconhecido</string>
+    <string name="d_checkCredentials_errorMessage_notFound">API não encontrada</string>
+    <string name="d_checkCredentials_errorMessage_notFoundWithMessage">API não encontrada: %s</string>
+    <string name="d_checkCredentials_errorMessage_unknownWithMessage">Error desconhecido: %s</string>
+    <string name="d_testConnection_fail_title">Teste de conexão falhido</string>
+    <string name="dialog_title_downloadFileFormat">Selecione o formato do arquivo</string>
+    <string name="pref_desc_connection_url">Exemplos:[...]</string>
```

Comment:
I'd rather have it [with examples](https://github.com/wallabag/android-app/blob/master/app/src/main/res/values/strings.xml#L179). Not sure these are as relevant as they were, but it shouldn't hurt.